src/viewers/PictureInPictureViewer.js

```diff
@@ -44,15 +51,16 @@ const PictureInPictureViewer = props => {
     lensSelection = 0,
     lensRadius = 100,
     lensBorderColor = [255, 255, 255],
-    lensBorderRadius = 0.02
+    lensBorderRadius = 0.02,
+    clickCenter = true,
   } = props;
   const viewState =
     initialViewState || getDefaultInitialViewState(loader, { height, width });
-  const detailViewState = { ...viewState, id: 'detail' };
+  const detailViewState = { ...viewState, id: DETAIL_VIEW_ID };
   const detailView = new DetailView({
     initialViewState: detailViewState,
     height,
-    width
+    width,
```

Comment:
This comma was added by `prettier --write` and has nothing to do with my changes.